mgmt_cli_test.py

```diff
@@ -96,11 +96,62 @@ def test_manager_sanity(self):
         4) test_client_encryption
         :return:
         """
+        self.test_basic_backup()
         self.test_repair_multiple_keyspace_types()
         self.test_mgmt_cluster_crud()
         self.test_mgmt_cluster_healthcheck()
         self.test_client_encryption()
 
+    def update_cred_file(self):
+        # FIXME: add to the nodes not in the same region as the bucket the bucket's region
+        # this is a temporary fix, after https://github.com/scylladb/mermaid/issues/1456 is fixed, this is not necessary
+        cred_file = '/etc/scylla-manager-agent/scylla-manager-agent.yaml'
+        region = self.params.get('region_name').split(' ')[0]
+        for node in self.db_cluster.nodes:
+            node.remoter.run(f'sudo chmod o+w {cred_file}')
+            node.remoter.run(f"sudo echo -e \"s3:\n   region: {region}\" >> {cred_file}")
+            node.remoter.run('sudo systemctl restart scylla-manager-agent')
+
+    def create_ks_and_tables(self, num_ks, num_table):
```

Comment:
`create_ks_and_tables_for_backup`  might be a bit clearer ?

Reply:
it could be used not only for backups, as this is planned to be a function that will generate data for either backup, restore (that still not implemented in the mgmt), repair or anything else...

Reply:
like the create_c1c2_table function in dtest